Tests/DependencyInjection/Factory/Propel1StorageFactoryTest.php

```diff
@@ -3,8 +3,6 @@
 namespace Payum\Bundle\PayumBundle\Tests\DependencyInjection\Factory;
```

Comment:
Not for this PR, but we dropped support for Propel entirely, so all these classes can be removed.

Reply:
I can do it in this PR or in seperate later. Just let me know